tests/basic/pool_function_test.go

```diff
@@ -535,3 +535,143 @@ var _ = Describe("{PoolExpandWhileResizeDiskInProgress}", func() {
 	})
 
 })
+
+var _ = Describe("{PoolExpandResizeDiskInMaintenanceMode}", func() {
+	var testrailID = 34542861
+	// testrailID corresponds to: https://portworx.testrail.net/index.php?/tests/view/34542861
+
+	/*
+		Steps:
+			1. Move a node to maintenance mode.
+			2. Initiate pool expand with resize-disk operation.
+			3. Exit out of maintenance mode (PX only performs pool expand in normal mode, not in maintenance mode)
+			4. Verify pool expand operation goes to completion.
+	*/
+
+	BeforeEach(func() {
+		contexts = scheduleApps()
+		StartTorpedoTest("PoolExpandResizeDiskInMaintenanceMode",
+			"Initiate pool expand with resize-disk when node is already in maintenance mode", nil, testrailID)
+	})
+
+	JustBeforeEach(func() {
+		poolIDToResize = pickPoolToResize()
+		log.Infof("Picked pool %s to resize", poolIDToResize)
+		poolToResize = getStoragePool(poolIDToResize)
+	})
+
+	JustAfterEach(func() {
+		AfterEachTest(contexts)
+	})
+
+	AfterEach(func() {
+		appsValidateAndDestroy(contexts)
+		EndTorpedoTest()
+	})
+
+	It("Start pool expand with resize-disk on node which is already in maintenance mode ", func() {
```

Comment:
can you push these steps also to aetos using log.infoD

Reply:
Done